pyglet/canvas/base.py

```diff
@@ -183,14 +183,13 @@ def get_best_config(self, template=None):
         """
         configs = None
         if template is None:
-            for template_config in [
-                gl.Config(double_buffer=True, depth_size=24),
-                gl.Config(double_buffer=True, depth_size=16),
-                None]:
+            for template_config in [gl.Config(double_buffer=True, depth_size=24),
+                                    gl.Config(double_buffer=True, depth_size=16),
+                                    None]:
```

Comment:
I actually prefer how it was before, this style forces things so far out to the right (but this isn't a very bad example). Just my opinion.